src/App.js

```diff
@@ -1,13 +1,23 @@
 import './App.css';
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
-import Home from './components/Home';
+// import Home from './components/Home';
+import Login from './components/Login/Login';
+import Dashboard from './components/Dashboard/dashboard';
 
 function App() {
 	const router = createBrowserRouter([
+		// {
```

Comment:
Why is this code commented?
Are you going to uncomment it later? if this is the case please add a _TODO: WIP (Work in progress)_ comment to know this will be useful later

`// TODO: WIP {`